audformat/core/utils.py

```diff
@@ -1399,6 +1400,16 @@ def to_filewise_index(
         ValueError: if ``output_folder`` contained in path to files of
             original data
 
+    Examples:
+        >>> index = segmented_index(
+        ...     files=['f.wav', 'f.wav'],
+        ...     starts=[0, 0.5],
+        ...     ends=[0.5, 1],
+        ... )
+        >>> # Split into single files
```

Comment:
do we really need this comment?

Reply:
I thought as a reminder that this function does not only provide an index, but does also create files.

But I'm also fine with removing it.

Reply:
I think it's clear enough

Reply:
I removed it.